server/src/test-fast/java/com/thoughtworks/go/config/update/CreateOrUpdateDefaultJobTimeoutCommandTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.thoughtworks.go.config.update;
+
+import com.thoughtworks.go.config.BasicCruiseConfig;
+import com.thoughtworks.go.config.exceptions.GoConfigInvalidException;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+public class CreateOrUpdateDefaultJobTimeoutCommandTest {
+
+    private BasicCruiseConfig cruiseConfig = new BasicCruiseConfig();
+
+    @Test
+    void shouldUpdateTheDefaultJobTimeout() throws Exception {
+        String defaultJobTimeout = "10";
+        CreateOrUpdateDefaultJobTimeoutCommand command = new CreateOrUpdateDefaultJobTimeoutCommand(defaultJobTimeout);
+        command.update(cruiseConfig);
+
+        assertThat(cruiseConfig.server().getJobTimeout()).isSameAs(defaultJobTimeout);
+    }
+
+    @Test
+    void shouldReturnTrueWhenTheDefaultJobTimeoutIsValid() {
+        String defaultJobTimeout = "10";
+        CreateOrUpdateDefaultJobTimeoutCommand command = new CreateOrUpdateDefaultJobTimeoutCommand(defaultJobTimeout);
+
+        assertThat(command.isValid(cruiseConfig)).isTrue();
```

Comment:
We should call update on the cruise-config prior to calling isValid.

Not calling the update command has no effect on the cruise-config and will always be valid.